.dockerignore

```diff
@@ -1,13 +1,15 @@
 node_modules
 
-**/.git
```

Comment:
Apparently, Sentry needs git to execute `release` logic smoothly (ex: infer release name, link commits to a release....)